src/dotlottie-player.ts

```diff
@@ -302,37 +231,346 @@ export class DotLottiePlayer extends LitElement {
     }
   }
 
-  private _configureDefaultLottie(id: string) {
-    const idx = Number(id);
+  private async _fetchDotLottie(url: string): Promise<{ animations: AnimationItem[]; manifest: Manifest | undefined }> {
+    const data = await fetch(url, {
+      method: 'GET',
+      mode: 'cors',
+      headers: {
+        'Response-Type': 'arraybuffer',
+      },
+    })
+      .then((buffer) => {
+        return buffer.arrayBuffer();
+      })
+      .then((buffer) => {
+        const animations: AnimationItem[] = [];
+        const animAndManifest: { animations: AnimationItem[]; manifest: Manifest | undefined } = {
+          animations: [],
+          manifest: undefined,
+        };
+
+        const data = unzipSync(new Uint8Array(buffer));
```

Comment:
why not using `unzip` instead, i think `unzipSync` is synchronize and will block the main thread

Reply:
@theashraf I'll fix that in a separate PR